### PR TITLE
apt*: add French translation

### PR DESCRIPTION
**Pull request by bbonnin**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.fr/linux/apt-file.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Mise à jour la base de données des métadonnées:
+
+`sudo apt update`
+
+- Recherche des paquets qui contiennent le fichier ou le chemin d'accès spécifié:
+
+`apt-file search {{part/of/filename}}`
+
+- Énumére le contenu d'un paquet spécifique:
```
Comment (Linkid) [inline pages.fr/linux/apt-file.md:13]: It is « Énum**è**re » ;)

**Review by Linkid — CHANGES_REQUESTED** [review]
:+1:

**Review by schneiderl — APPROVED** [review]
Do not know the language but syntax-wise it looks good.

File: pages.fr/linux/apt-file.md (hunk 1 above)
Comment (bbonnin) [reply pages.fr/linux/apt-file.md]: Oops... Fixed.

**Pull request merged**